router/dns.go

```
@@ -61,7 +61,7 @@ func (d *DNS) resolveConvox(w dns.ResponseWriter, r *dns.Msg) {
 		for _, q := range m.Question {
 			switch q.Qtype {
 			case dns.TypeA:
-				if ep, ok := d.router.endpoints[strings.TrimSuffix(q.Name, ".")]; ok {
+				if ep, _ := d.router.matchEndpoint(strings.TrimSuffix(q.Name, ".")); ep != nil {
```

Safe to discard the error? Seems OK as there is a nil check on ep.